backend/app/api/endpoints/chapters.py

```
@@ -0,0 +1,592 @@
+            "success": True,
+        }
+
+
+# Enhanced Chapter Metadata and Tab Management Endpoints
+
+
+@router.patch("/{book_id}/chapters/bulk-status", response_model=dict)
+async def update_chapter_status_bulk(
+    book_id: str,
+    update_data: BulkStatusUpdate,
+    current_user: Dict = Depends(get_current_user_from_session),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)